node/src/blockchain/blockchain_interface_utils.rs

```diff
@@ -0,0 +1,1412 @@
+            chain,
+            web3_batch,
+            consuming_wallet.clone(),
+            gas_price.clone(),
+            pending_nonce,
+            new_fingerprints_recipient,
+            accounts.clone(),
+        )
+        .wait();
+
+        System::current().stop();
+        system.run();
+        let tlh = TestLogHandler::new();
+        let timestamp_after = SystemTime::now();
+        let recording_result = accountant_recording.lock().unwrap();
+        let processed_payments = result.unwrap();
+        let message = recording_result.get_record::<PendingPayableFingerprintSeeds>(0);
+        assert_eq!(recording_result.len(), 1);
+        assert!(timestamp_before <= message.batch_wide_timestamp);
+        assert!(timestamp_after >= message.batch_wide_timestamp);
+        assert_eq!(
+            message.hashes_and_balances,
+            vec![
+                HashAndAmount {
+                    hash: H256::from_str(
+                        "35f42b260f090a559e8b456718d9c91a9da0f234ed0a129b9d5c4813b6615af4"
+                    )
+                    .unwrap(),
+                    amount: accounts_1.balance_wei
+                },
+                HashAndAmount {
+                    hash: H256::from_str(
+                        "7f3221109e4f1de8ba1f7cd358aab340ecca872a1456cb1b4f59ca33d3e22ee3"
+                    )
+                    .unwrap(),
+                    amount: accounts_2.balance_wei
+                },
+            ]
+        );
+        assert_eq!(
+            processed_payments[0],
+            ProcessedPayableFallible::Correct(PendingPayable {
+                recipient_wallet: accounts_1.wallet,
+                hash: H256::from_str(
+                    "35f42b260f090a559e8b456718d9c91a9da0f234ed0a129b9d5c4813b6615af4"
+                )
+                .unwrap()
+            })
+        );
+        assert_eq!(processed_payments[1], ProcessedPayableFallible::Failed(RpcPayableFailure{
+            rpc_error: Rpc(Error {
+                code: ServerError(429),
+                message: "The requests per second (RPS) of your requests are higher than your plan allows.".to_string(),
+                data: None,
+            }),
+            recipient_wallet: accounts_2.wallet,
+            hash: H256::from_str("7f3221109e4f1de8ba1f7cd358aab340ecca872a1456cb1b4f59ca33d3e22ee3").unwrap(),
+        }));
+        tlh.exists_log_containing(
+            &format!("DEBUG: {test_name}: Common attributes of payables to be transacted: sender wallet: {}, contract: {:?}, chain_id: {}, gas_price: {}",
+                     consuming_wallet,
+                     chain.rec().contract,
+                     chain.rec().num_chain_id,
+                     gas_price
+            )
+        );
+        tlh.exists_log_containing(&format!(
+            "INFO: {test_name}: {}",
+            transmission_log(chain, &accounts, gas_price)
+        ));
+    }
+
+    #[test]
+    fn sign_transaction_just_works() {
```

Comment:
This test also only uses the same perspective of looking at the subject, asserting only a tx hash string. If you repeat such an assertion a dozen times it really is a waste, as you don't end up wiser about whether all the components in the interior do the right things. And unfortunately, I don't think you can really do better. If you still had had the BatchPaybleTools, then yes you could've intercept partial results and pull them out from where they came to creation, just to be asserted on separately. 

But the way it is presented to me here, I don't think we want these test. It's already tested with the same quality by a different test I've already checked above.